fibonacci-without-recursive.js

```diff
@@ -0,0 +1,23 @@
+// Napisz funkcję fibonacci(n), która, nie używając rekurencji, zwróci n-ty wyraz Ciągu Fibonacciego.
+
+function fibonacci(n) {
+    if (n < 2) {
+        return n;
+    }
+
+    let a = 0,
+        b = 1,
+        c = 1;
```

Comment:
Before you write something like this, you should remove the line and check if anything has changed. I can guarantee you that you don't need this line. Please, remove it and try to understand why you don't need it.